nncf/quantization/algorithms/weight_compression/awq_patterns.py

```diff
@@ -29,9 +29,49 @@ def create_matmul_mul_matmul(matmul_metatype, multiply_metatype) -> GraphPattern
     return pattern
 
 
-def get_awq_patterns(matmul_metatype, multiply_metatype):
+def matmul_mul_operation(matmul_metatype, multiply_metatype) -> GraphPattern:
+    pattern = GraphPattern()
+    linear_node = pattern.add_node(**{GraphPattern.LABEL_ATTR: "LINEAR", GraphPattern.METATYPE_ATTR: matmul_metatype})
+    mul_node = pattern.add_node(**{GraphPattern.LABEL_ATTR: "MULTIPLY", GraphPattern.METATYPE_ATTR: multiply_metatype})
+    pattern.add_edge(mul_node, linear_node)
+
+    return pattern
+
+
+def matmul_operation(matmul_metatype) -> GraphPattern:
+    pattern = GraphPattern()
+    pattern.add_node(**{GraphPattern.LABEL_ATTR: "LINEAR", GraphPattern.METATYPE_ATTR: matmul_metatype})
+
+    return pattern
+
+
+def atomic_activations_operations_pattern(atomic_activations_operations) -> GraphPattern:
+    pattern = GraphPattern()
+    pattern.add_node(
+        **{GraphPattern.METATYPE_ATTR: atomic_activations_operations, GraphPattern.LABEL_ATTR: "ATOMIC_ACTIVATIONS"}
+    )
+    return pattern
+
+
+@AWQ_PATTERNS.register("ACT_Mul_MatMul")
+def create_linear_mul_activations(matmul_metatype, multiply_metatype, atomic_activations_operations) -> GraphPattern:
+    linear = matmul_mul_operation(matmul_metatype, multiply_metatype)
+    activations = atomic_activations_operations_pattern(atomic_activations_operations)
+    activations.join_patterns(linear)
+    return activations
+
+
+@AWQ_PATTERNS.register("ACT_MatMul")
```

Comment:
The same about the name here. Maybe `MatMul_Act` is more correct?

Reply:
Done